### PR TITLE
Fix Safari support for CSS list-style-position property

### LINKED ISSUE #18934: css.properties.list-style-position - Incorrect support status for Safari
### What type of issue is this?

Incorrect support data (ex. Chrome says "86" but support was added in "40")

### What information was incorrect, unhelpful, or incomplete?

The last paragraph in the "Try it" [section](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-position#try_it) of the MDN page claims different behaviors across browsers for `list-style-position: inside` with a link pointing to [Mozilla Bug 36854](https://bugzilla.mozilla.org/show_bug.cgi?id=36854), but the last [comment](https://bugzilla.mozilla.org/show_bug.cgi?id=36854#c38) in the bug contains updated info in this regard.

To summarize, the spec says that

>The marker box is generated by the ::marker pseudo-element of a list item as the list item’s first child[…]

So if a block element is placed first in a list element, the marker should be placed on the fist line and the block element on the second line. The conformance of browsers are:

 - Firefox, Opera, Internet Explorer: conforming (according to the MDN page);
 - Chrome: conforming (according to the last comment of the Mozilla Bug);
 - Safari: ~non-conforming / unclear (according to the MDN page and the [Webkit Bug 227000](https://bugs.webkit.org/show_bug.cgi?id=227000))~ See the comment below.

### What browsers does this problem apply to, if applicable?

Safari

### What did you expect to see?

~If Safari is confirmed to not conform in the mentioned cases, its support status should be changed to "partial" with an implementation note.~ The support status for Safari be updated.

### Did you test this? If so, how?

No available MacOS device to test.

### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

MDN page: https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-position
Mozilla Bug 36854: https://bugzilla.mozilla.org/show_bug.cgi?id=36854
WebKit Bug 227000: https://bugs.webkit.org/show_bug.cgi?id=227000

### Do you have anything more you want to share?

Related MDN content issue: https://github.com/mdn/content/issues/24502

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-position

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `css.properties.list-style-position`
* Report started: 2023-02-17T00:33:41.036Z

</details>

### PR DESCRIPTION
**Pull request by Tunzeki**

### CODE REVIEW

**Comment by queengooborg** [comment]
Hi @Tunzeki, thank you for your PR and welcome to BCD!

Would you mind sharing the source of your information, and how you tested to confirm it is accurate information?

**Comment by Tunzeki** [comment]
> Hi @Tunzeki, thank you for your PR and welcome to BCD!
> 
> Would you mind sharing the source of your information, and how you tested to confirm it is accurate information?

The second paragraph under **Description** on the MDN web docs page on `list-style-position` (https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-position) states:

"If a block element is the first child of a list element declared as list-style-position: inside, then the block element is placed on the line after the marker-box."

I tested this on Chrome 109 and it checks out. 

However, Safari places the block element on the same line as the marker-box. This was discussed in a related issue now closed (https://github.com/mdn/content/issues/24502#issuecomment-1483132768). One commenter pointed out that they tested it on Safari 16 and noticed the difference in behaviour (they attached a screenshot to their comment, which I have linked to). I also tested it on a friend's Safari 16 and can confirm that it is accurate information.

File: css/properties/list-style-position.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
             },
             "opera_android": "mirror",
             "safari": {
-              "version_added": "1"
+              "version_added": "1",
+              "notes": "In Safari, if a block element is the first child of a list element declared as `list-style-position: inside`, then the marker box is placed on the same line as the block element."
```
Comment (queengooborg) [inline css/properties/list-style-position.json:30]:
```suggestion
              "notes": "In Safari, if a block element is the first child of a list element declared as <code>list-style-position: inside</code>, then the marker box is placed on the same line as the block element."
```

**Review by queengooborg — APPROVED** [review]
Thank you for following up with information, and apologies for taking so long to review this.  I trust your research, just one quick change and this will be good to go!

**Pull request merged**